docs/source/reading_plans.rst

```diff
@@ -1,123 +1,237 @@
+It is possible to skip days. For example, if you do not wish to have a reading on the
+weekend:
 
-1=Jan-1;Gen.1-2,Ps.1-2,Matt.1-2
-2=Jan-2;Gen.3-4,Ps.3-5,Matt.3-4
-3=Jan-3;Gen.5-6,Ps.6-8,Matt.5
-...
-146=May-26;Jos 12,Isa 16,2Tim 2
-...
-334=Nov-30;Est 7-8,Oba,Heb 3-5
+.. code-block::
 
-As mentioned in the ".conf file" section, you will need to add this line to the .conf file: AndBibleReadingPlanDateBased=true
+    1=Gen.1, Matt.1
+    3=Gen.2, Matt.2
 
-OSIS format references must be used and OSIS Bible book names must be used. These will be translated to the current language when displayed in AndBible. E.g. Gen.1,Matt.1-Matt.2,Ps.119.1-Ps.119.10
+You can also add a date-based reading plan, which includes the month and day
+number for each day. The date-based plan will always open at today's date by
+default (The regular plan opens the current day, which is the last day you haven'take
+yet completed). For a date-based plan, the :code:`.properties` file must always
+begin the readings at :code:`1=`, and the date must be in the format of 3 letters
+for the month, then a dash and then a month day number followed by semicolon (;).
+For example:
 
-Example reading plans can be found here.
+.. code-block::
 
-A full list of OSIS book names can be found here
+    1=Jan-1;Gen.1-2,Ps.1-2,Matt.1-2
+    2=Jan-2;Gen.3-4,Ps.3-5,Matt.3-4
+    3=Jan-3;Gen.5-6,Ps.6-8,Matt.5
+    ...
+    146=May-26;Jos 12,Isa 16,2Tim 2
+    ...
+    334=Nov-30;Est 7-8,Oba,Heb 3-5
 
-Deuterocanonical Books
-----------------------------
-The default versification for reading plans is KJV. If you wish to include deuterocanonical books in your plan you must specify the versification in the properties file. However, this is not supported for the new zip format added in 4.0. See the ".conf file" section above for how to do this for zip format. E.g.:
+As mentioned in `the .conf file`_ section, you must add the following line to
+the :code:`.conf` file when using a date-based plan:
 
-Versification=Vulg
-1=Sir.1-Sir.2
-2=1Macc.1-1Macc.2
-3=Bar.1-Bar.2
+:code:`AndBibleReadingPlanDateBased=true`
 
-Set Start to Another Day
--------------------
-If you started the readings on a previous day and have just switched to AndBible then select Menu from the Daily Reading screen and then select 'Set Start Date...'. You may then jump to 'Move the current day to a specific day' below to skip over readings you have already done.
+Example reading plans can be found
+`here <https://github.com/AndBible/and-bible/tree/master/app/src/main/assets/readingplan>`_.
```

Comment:
"Branch not found, redirected to default branch."

current-stable is the one that corresponds to earlier master branch